### PR TITLE
Pushover.net

### PR DESCRIPTION
**Pull request by zakwilson**
This adds initial support for Pushover

### CODE REVIEW

File: test/pushover_test.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    svc = service(:logs, {:pushover_app_token => 'a sample token',
+                          :pushover_user_token => 'a different token'},
+                  payload)
+
+    http_stubs.post '/services/hooks/incoming-webhook' do |env|
+      [200, {}, '']
+    end
+
+    # svc.receive_logs
+    # TODO - stub the actual post
```
Comment (troy) [inline test/pushover_test.rb]: Can you add this?

Comment (zakwilson) [inline test/pushover_test.rb]: Yes. That and the 1kb limit are coming after I've had lunch.

File: test/pushover_test.rb [hunk 2]
```diff
@@ -0,0 +1,40 @@
+require File.expand_path('../helper', __FILE__)
+
+require 'fakeweb'
+
+class PushoverTest < PapertrailServices::TestCase
+
+  def setup
+    FakeWeb.register_uri(:post, "https://api.pushover.net/1/messages.json",
+                         :body => { :status => 1 }.to_json,
+                         :content_type => "application/json")
```
Comment (eric) [inline test/pushover_test.rb]: using `FakeWeb` shouldn't be necessary. The `http_stubs.post` code below is intended to be the mocking mechanism.

```
    http_stubs.post '/1/messages.json' do |env|
      [200, {}, { :status => 1 }.to_json]
    end
```

**Comment by troy** [comment]
In production, the 2 user-provided values will be `token` and `user_key` to match other services. I'll change them when I merge if you don't see this and change them first.

**Comment by troy** [comment]
@zakwilson would you mind `bundle install`-ing and removing the `Zapier` class from this PR, then pushing? I would but can't do it until after I merge into master. No hurry.

**Comment by troy** [comment]
Closing this PR and re-submitting a new PR with these 2 fixes.

**Pull request closed without merging**